andela/hc-shleemypants
# add FAQs section

**What does this PR do ?**
Adds FAQs section to healthchecks 

**Description of tasks to be completed**
As a new user interested in getting onboard, then I should :

- find FAQs section to help ease my process of getting onboard and answer some common questions that may arise.
- Find a live chat widget.

**How should this be manually tested?**
Pull the latest changes, run the ap. Head to the browser, there should be a FAQs tab.

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

- 155965544

**Screenshots**

<img width="1161" alt="faqs" src="https://user-images.githubusercontent.com/17777137/38484702-1a14bce6-3be0-11e8-885c-0491c4a814a3.png">
